vtex-sites/nextjs.store
# Feat: FSUI `Overlay` component

## What's the purpose of this pull request?

Aims to use the `Overlay` component from the new architecture: faststore/ui -> faststore/components.
It also removes the global overlay styles that were coupled with the `SlideOver` component.

## How does it work?

Adapts the store to use the new component.

## How to test it?

The Pages should show the Overlay (`SlideOver`) just as we can see previously.

Starter PR
https://github.com/vtex-sites/nextjs.store/pull/311
